internal/http/services/owncloud/ocgraph/conversions.go

```diff
@@ -429,6 +431,114 @@ func (s *svc) cs3ShareToDriveItem(ctx context.Context, info *provider.ResourceIn
 	return d, nil
 }
 
+func (s *svc) OCMReceivedShareToDriveItem(ctx context.Context, receivedOCMShare *ocm.ReceivedShare) (*libregraph.DriveItem, error) {
+
+	createdTime := utils.TSToTime(receivedOCMShare.Ctime)
+
+	creator, err := s.getUserByID(ctx, receivedOCMShare.Creator)
+	if err != nil {
+		return nil, err
+	}
+
+	grantee, err := s.cs3GranteeToSharePointIdentitySet(ctx, receivedOCMShare.Grantee)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Debug().Interface("receivedOCMShare", receivedOCMShare).Msg("processing received OCM share")
+
+	var webdav_uri string = ""
```

Comment:
nit: no need for `= ""`, `""` is the default value for strings in Go

And if you remove this, you can do 
```
var webdav_uri, webapp_uri, shared_secret string
```